python/sdk/test/model_test.py

```diff
@@ -29,15 +29,20 @@
 from merlin.model import ModelType
 from merlin.protocol import Protocol
 from urllib3_mock import Responses
+from client.models import GPU
 
 import merlin
 from merlin import AutoscalingPolicy, DeploymentMode, MetricsType
 
 responses = Responses('requests.packages.urllib3')
 
 default_resource_request = cl.ResourceRequest(1, 1, "100m", "128Mi")
+gpu = GPU(resource_type="nvidia.com/gpu", node_selector={"cloud.google.com/gke-accelerator": "nvidia-tesla-p4"})
```

Comment:
Can we use `cl.GPU` to keep it uniform? Then we can remove `from client.models import GPU` above as well.